XIVComboExpanded/Combos/SAM.cs

```diff
@@ -296,6 +297,12 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                         return SAM.HissatsuGuren;
                 }
             }
+
+            if (IsEnabled(CustomComboPreset.SamuraiShintenZanshinFeature))
+            {
+                if (level >= SAM.Levels.Zanshin && HasEffect(SAM.Buffs.ZanshinReady))
```

Comment:
SAM.Levels and SAM.Buffs were already updated, so only the action ID `SAM.Zanshin` needed to be added.